guillotina/commands/server.py

```diff
@@ -21,33 +21,37 @@ def get_parser(self):
         parser.add_argument("--asgi-server", default="uvicorn", type=str)
         return parser
 
-    def run(self, arguments, settings, app):
-        port = arguments.port or settings.get("address", settings.get("port"))
-        host = arguments.host or settings.get("host", "0.0.0.0")
+    async def run(self, arguments, settings, app):
+        host = arguments.host or app.settings.get("host", "0.0.0.0")
```

Comment:
This breaks all actual configs!

Reply:
which thing? I just swiped the lines

Reply:
app.settings is mapped to server_settings, isn't it?

Reply:
not, it's mapped to app_settings (after being merge with all the configs provided by the user) https://github.com/plone/guillotina/pull/781/files#diff-9aee2a99444a30b44f1a3a9ce2b1d76aR264

Reply:
Sorry confused with typing